### PR TITLE
fan page 

### PR DESCRIPTION
**Pull request by bskassama**

### CODE REVIEW

**Comment by bskassama** [comment]
I have already completed Module 2.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <li> The Vikings have a new stadium to be proud of.</li>
+    <li>The Vikings have the best running back in the NFL.</li><li> The Vikings have the best Quaterback in the NFL.</li><li>The Viking fans are the best in the NFl.</li>
+    <li>Minnesota deserves a championship this year.</li>
+    </ul>
+    
+    
+   
+    
+    <footer>
+    <small><b>Copyright &copy; 2016 Bakary Kassama &nbsp; email:  bakary.kassama@bkassama.com</b></small></footer>
```
Comment (mbMosman) [inline index.html]: You'll want to pull out the non-semantic HTML for the small & bold, you can add that in the next part w/ CSS.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Minnesota Vikings</title>
+<meta charset="utf-8">
+</head>
+<body>
+   <h1>Minnesota Vikings Football Team</h1>
```
Comment (mbMosman) [inline index.html]: You should use the `<header>` element with your `<h1>`.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+<html lang="en">
+<head>
+<title>Minnesota Vikings</title>
+<meta charset="utf-8">
+</head>
+<body>
+   <h1>Minnesota Vikings Football Team</h1> 
+    <nav> <a href="http://www.vikings.com">Vikings Homepage</a> &nbsp; <a href="http://www.NFL.com">NFL Homepage </a>&nbsp; 
+        <a href="http://www.vikings.com/schedule/season-schedule.html"> Vikings Schedule</a></nav>
+    <h3> My Favorite Team</h3>
```
Comment (mbMosman) [inline index.html]: You should put your main content inside of a `<main>` element.

**Pull request closed without merging**